src/components/CountryDetails.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+// import countries from '../countries.json'
+import { Link } from 'react-router-dom'
+
+export default function CountryDetails(props) {
+    const countryId = props.match.params.id // "id" use the same name as in App.js line 18
+
+    const country = props.countries.find(element => element.cca3 === countryId);
+
+    const borders = country.borders.map(border => {
+        let borderCountry = props.countries.find(country => country.cca3 === border)
```

Comment:
Wouldn't a const be more fitting here than a let? Will the variable be reassigned?